core/validatorapi/router_internal_test.go

```diff
@@ -962,6 +937,15 @@ func testRawRouter(t *testing.T, handler testHandler, callback func(context.Cont
 	callback(context.Background(), server.URL)
 }
 
+// newTestHandler returns a new testHandler with NodeVersionFunc set.
+func newTestHandler() testHandler {
+	return testHandler{
+		NodeVersionFunc: func(ctx context.Context) (string, error) {
```

Comment:
why is this required?

Reply:
Since this endpoint is required for go-eth2-client to initialize - https://github.com/attestantio/go-eth2-client/blob/master/http/service.go#L142

Reply:
Suggest adding it a method on `testHandler` and removing the `/eth/v1/node/version` endpoint from `newBeaconHandler` since it isn't used anymore.
```
func (h testHandler) NodeVersion(ctx) string, err {
  if h.NodeVersionFunc != nil {
    return h.NodeVersionFunc(ctx)
  }
  // Default to 
  return "mock_version", nil
}
```